app/views/accessibility/show.html.erb

```diff
@@ -0,0 +1,94 @@
+<%= content_for :title, page_title(:accessibility) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t('page_titles.accessibility') %></h1>
```

Comment:
If we have more pages like this, might be worth adding it as a markdown template. You can render it directly using [redcarpet](https://github.com/vmg/redcarpet).

Reply:
That’d be awesome @tijmenb! Currently, pages that are quite content heavy include:

* Terms of use
* Privacy statement
* Accessibility statement
* List of opted-in providers

Reply:
@tijmenb and @paulrobertlloyd Should we merge this and seek to do that in another PR?

Reply:
Totes down with doing that in a separate PR 🙇